lib/elegant_alert_dialog.dart

```diff
@@ -1,186 +1,2 @@
-    this.title,
-    this.deleteButtonColor = Colors.blue,
-    this.deleteButtonText = 'Confirm',
-    this.onDeletePressed,
-    this.cancelButtonColor = Colors.grey,
-    this.cancelButtonText = 'Learn more',
-    this.onCancelPressed,
-  }) {
-    borderColor = errorColor;
-    backgroundColor = Colors.white;
-    radius = 5;
-    elegantAlertType = ElegantAlertDeleteType(
-      cancelButtonColor: cancelButtonColor!,
-      cancelButtonText: cancelButtonText!,
-      onCancelPressed: onCancelPressed,
-      deleteButtonColor: deleteButtonColor!,
-      deleteButtonText: deleteButtonText!,
-      onDeletePressed: onDeletePressed,
-    );
-  }
-
-  ElegantAlertDialog.info({
-    super.key,
-    required this.description,
-    this.title,
-    this.confirmButtonColor = Colors.blue,
-    this.confirmButtonText = 'Confirm',
-    this.onConfirmButtonPressed,
-  }) {
-    borderColor = infoColor;
-    backgroundColor = Colors.white;
-    radius = 5;
-    elegantAlertType = ElegantAlertInfoType(
-      confirmButtonColor: confirmButtonColor!,
-      confirmButtonPressed: onConfirmButtonPressed,
-      confirmButtonText: confirmButtonText!,
-    );
-  }
-
-  final Widget? title;
-  final Widget description;
-  late Color backgroundColor;
-  late Color? borderColor;
-  late double radius;
-  late List<Widget> actions;
-
-  //! multi actions constructor
-  String? primaryButtonText;
-  String? secondButtonText;
-  String? customButtonText;
-
-  Function()? onPrimaryButtonPressed;
-  Function()? onSecondaryButtonPressed;
-  Function()? onCustomButtonPressed;
-
-  Color? primaryButtonColor;
-  Color? secondButtonColor;
-  Color? customButtonColor;
-
-  //! permission constructor
-  String? allowButtonText;
-  String? denyButtonText;
-
-  Function()? onAllowButtonPressed;
-  Function()? onDenyButtonPressed;
-
-  Color? allowButtonColor;
-  Color? denyButtonColor;
-
-  //! info constructor
-  String? confirmButtonText;
-  Function()? onConfirmButtonPressed;
-  Color? confirmButtonColor;
-
-  //! delete constructor
-  String? deleteButtonText;
-  String? cancelButtonText;
-
-  Function()? onDeletePressed;
-  Function()? onCancelPressed;
-
-  Color? deleteButtonColor;
-  Color? cancelButtonColor;
-
-  late ElegantAlertType? elegantAlertType;
-
-  void show(BuildContext context) {
-    showDialog(
-      context: context,
-      builder: (context) {
-        return Dialog(
-          child: this,
-        );
-      },
-    );
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    return Container(
-      height: 250,
-      decoration: BoxDecoration(
-        color: backgroundColor,
-        borderRadius: BorderRadius.circular(10),
-        border: Border.all(
-          width: 1,
-          color: borderColor!,
-        ),
-      ),
-      child: elegantAlertType?.build(),
-    );
-  }
-}
+export 'package:elegant_alert_dialog/widgets/elegant_alert_dialog.dart';
+export 'package:elegant_alert_dialog/widgets/support_widgets/elegant_body_widet.dart';
```

Comment:
wrong typo missing "g" in `elegant_body_widet`;

Reply:
yup will change